front-end/src/components/TranscriptUploadModal.jsx

```diff
@@ -22,6 +24,7 @@ class TranscriptUploadModal extends React.Component {
   handleData(result) {
     if (!this.validateData(result)) {
       this.setState({isFileValid: false});
+      store.dispatch(disableAnalysisButton())
```

Comment:
What if the user already has a valid transcript uploaded? I don't think we should disable in that case (since we have not yet successfully sent data to the BE).

Reply:
If the user has uploaded a transcript and then clicks the upload button, IMO it 100% should not disable the analysis page until they have tried uploading a new transcript.
@helias22 @emilylwan @williamyao27 what do you all think?

Reply:
yeah this doesn't disable the button until they try to upload a new transcript. but if they do and the file is invalid then the button is disabled. if they just open and close the upload modal nothing will happen

Reply:
I'm not sure if the user uploads an invalid file that should disable the button either because the user might choose to just go and look at the old transcript again.

Reply:
okk